### DISCUSSION TITLE
RTMP stream without internet access

### DISCUSSION THREAD

**Discussion by MissingName01** [gopro/OpenGoPro#348] [Q&A]
I havent tried using this yet but would it work for my scenario?
as it is ive been messing around with my gopro and i can connect it to my wifi if my LAN has internet connection but i want it to be able to work and stream to my private server if there is no internet, when i try and connect to wifi and have my ISP unplugged my gopro wont even connect.

**Comment by martincarapia** [comment]
If you have a rtmp server on a private server that is connected to the same LAN as the camera then you should be able to access it. As long as there's an access point and the devices can communicate locally it's not a problem. Just make sure when you're defining the stream link that you use the local IP address of the machine and not localhost.
